irods_specific_implementation/CMakeLists.txt

```diff
@@ -81,6 +83,8 @@ set(CPACK_RPM_PACKAGE_VENDOR "iRODS Consortium")
 set(CPACK_RPM_PACKAGE_URL "https://irods.org")
 set(CPACK_RPM_PACKAGE_AUTOREQ 0)
 set(CPACK_RPM_PACKAGE_AUTOPROV 0)
+set(CPACK_RPM_PACKAGE_OBSOLETES ${IRODS_PACKAGE_NAME})
```

Comment:
or this one

Reply:
I don't see this line in the other repos I checked either. We're probably good to nix it.

Reply:
Will remove

Reply:
done